templates/documentation/runbooks/pod-restarts.md.tpl

```diff
@@ -28,6 +38,63 @@ correct direction of the source of the problem. It may also be a useful exercise
 kubectl -n {{ .Config.Name }}--<bento> describe deployment {{ .Config.Name }}
 ```
 
+If a pod recently crashed the previous logs may still be available and viewable via:
+
+```shell
+kubectl -n {{ .Config.Name }}--<bento> logs --previous <pod name>
+```
+
+where `<pod name>` is the name of the pod that just recently started up after the crash
+
+<!--- Block(podRestartsPodState) -->
+{{ file.Block "podRestartsPodState" }}
+<!--- EndBlock(podRestartsPodState) -->
+
+### Datadog Dashboard and Logs
+
+The {{ .Config.Name }} service has a dashboard in Datadog: https://app.datadoghq.com/dashboard/i6z-jcn-azh/terraform-{{ .Config.Name }}
+
+Look for any anomalies in the dashboard.
+
+{{ camelcase .Config.Name }} logs are available via a search: https://app.datadoghq.com/logs/livetail?query=service%3A{{ .Config.Name }}%20environment%3Aproduction%20status%3Aerror&index=main&from_ts=1652033594145&to_ts=1652206394145&live=true. Modify the search parameters to match the bento and environment the alert was generated from.
+
+Look at the {{ camelcase .Config.Name }} logs for a log line like the following:
+
+```
+[Triggered on {kube_namespace:{ .Config.Name }}--<bento>}] { .Config.Name }} Pod Restarts > 0 last 30m
+```
+
+Modify the `<bento>` value to match the bento the issue is alerting from. Once you find that line, then look at the few minutes before that for logs that give any indication of unpredictable behavior from the service itself.
```

Comment:
suggestion: perhaps add the log line to the search criteria? Minus the bento-specific part. This would make it faster for the OCE to perform this action.

Reply:
Perhaps, however, I did it this way on purpose because I can't be 100% sure that the Pod Restarts log line will be the relevant and helpful log entry for all services, so seeing the log entry in context is useful. By jumping directly to the Pod Restarts log line some context may be ignored and the OCE may miss important clues.

Reply:
Maybe have a separate link for just those logs then?

Reply:
Hmmm.  The link for the Pod Restart logs, and the start of the updated verbiage is at: https://github.com/getoutreach/stencil-base/pull/36/files#diff-beeb7b5cd356b4e7237fc9211ecaa2dd6fffef3ac7e1a5a96731ea84e31b50c6R59

Reply:
Ah, I misread the new link.